### ISSUE TITLE
Request for Improved Functionality in LORA Trainer: Automatic Saving of LORA during training run and Ability to Export/Import Finetuning Settings

### ISSUE THREAD

**Issue by IIIIIIIllllllllIIIII** [oobabooga/text-generation-webui#994]
Over the past few days, I've been utilizing the trainer, and I must say, I'm extremely impressed with its performance. However, I do have a minor suggestion for improvement.

Background of my request:
Currently, when training stable diffusion models, the model is saved multiple times during the training process (which can be adjusted in the settings to save the model every few epochs).

Feature request:
It would be beneficial to include this feature in the text-webui-trainer as well, so the ongoing lora could be automatically saved every XXX epochs (or after 25% 50% 75% of the training... etc.).

This would enable better progress tracking and evaluation for each epoch, help with finding the best lora version for a training run, saving loras in case of a computer crash or shutdown, and other advantages. Although I was unable to locate mcmonkey4evas contact information, I hope to submit this suggestion here in the hopes that it may be of use and maybe even realized soon! :)

Another mini feature request:
An additional feature that would be helpful is the option to save and load finetuning settings in txt or json format. The current process of taking screenshots of settings and manually saving them in the lora folder can be quite cumbersome. This feature would streamline the process and save users time and effort.

Thanks guys

**Comment by Gitterman69** [comment]
A convenient feature to have in the trainer would be automatic model saving, especially in situations where the training runs for an extended period, say, 10 hours, but overfits the model. Saving the model at the 50% mark or after 5 hours of training would suffice, which could save considerable time and resources.

**Comment by nurena24** [comment]
yes, a loading feature is much needed. btw- what are ideal train settings for you? i'm trying to train on raw text document but getting sub par results.
